<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PersoPassGen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generiere</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Generiertes Passwort (wird automatisch in Zwischenablage übernommen)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="InputButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Neu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Löschen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="qtwidgetsapplication1.qrc">:/QtWidgetsApplication1/buergerserviceorg.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="NumCharButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Zeichen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SpecCharButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sonderzeichen</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>441</width>
      <height>311</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>441</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>49</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>38</number>
    </attribute>
    <column>
     <property name="text">
      <string>Accountname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Zeichen</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sonderzeichen</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOeffnen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionHilfe"/>
    <addaction name="actionUeber"/>
    <addaction name="actionZertifikat"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOeffnen">
   <property name="text">
    <string>Oeffnen</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionUeber">
   <property name="text">
    <string>Ueber</string>
   </property>
  </action>
  <action name="actionZertifikat">
   <property name="text">
    <string>Zertifikat</string>
   </property>
  </action>
  <action name="actionNumChars">
   <property name="text">
    <string>Anzahl Zeichen ändern</string>
   </property>
  </action>
  <action name="actionSpecChars">
   <property name="text">
    <string>Mögliche Sonderzeichen ändern</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtwidgetsapplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
